backend/models/user.mgmodel.ts

```diff
@@ -10,24 +10,46 @@ export interface User extends Document {
   role: Role;
 }
 
-const UserSchema: Schema = new Schema({
-  firstName: {
-    type: String,
-    required: true,
-  },
-  lastName: {
-    type: String,
-    required: true,
-  },
-  authId: {
-    type: String,
-    required: true,
-  },
-  role: {
-    type: String,
-    required: true,
-    enum: ["Administrator", "Facilitator", "Learner"],
+const baseOptions = {
+  discriminatorKey: "type",
```

Comment:
Hey actually... what if we have the `discriminatorKey` as `"role"` (the property that's already on User)?

By specifying a different discriminator key like `"type"`, we end up with the same thing showing up twice on facilitators and learners:
<img width="463" alt="Screenshot 2024-07-12 at 2 10 54 AM" src="https://github.com/user-attachments/assets/4dc098a9-67dc-45d6-a8bd-95391f151ee8">

The catch is that you'll have to add an `Administrator` discriminator as well (or else creating an admin will cause an error) - should be ezpz, same as what you did for facilitator/learner discriminator, just with no additional properties (for now anyway) :D